disasterinfosite/static/js/app.js

```diff
@@ -138,7 +138,7 @@ $( document ).ready(function() {
 
   function submitLocation(lat,lng) {
     // reload the page with the lat,lng
-    document.location =  encodeURI(document.location.hash + "?lat=" + lat + "&lng=" + lng + "&loc=" + location_query_text);
+    document.location =  encodeURI('/' + "?lat=" + lat + "&lng=" + lng + "&loc=" + location_query_text);
```

Comment:
When the site is deployed in a subdirectory, the leading `/` pops us out of that subdirectory, breaking the link.